adapters/discord/src/message.ts

```diff
@@ -300,7 +301,15 @@ export class DiscordMessageEncoder<C extends Context = Context> extends MessageE
     } else if (type === 'audio') {
       await this.sendAsset('file', attrs, {
         ...this.addition,
-        content: this.buffer.trim(),
+        content: '',
+        attachments: [
+          {
+            waveform: '', // base64 encoded bytearray representing a sampled waveform
+            id: '0',
```

Comment:
Is it okay to just use `'0'` here, I think it is supposed to be a snowflake?

Reply:
Reference https://discord.com/developers/docs/reference#uploading-files

Reply:
![image](https://github.com/satorijs/satori/assets/96647698/f91301ff-6330-4f6d-ae5e-a35e8032697b)
Here's an official example

Reply:
Wait, it should be 0 here, not '0'.